broker-plugin/amqp-connector/src/main/java/org/apache/activemq/artemis/integration/amqp/LinkInitiator.java

```diff
@@ -249,6 +258,8 @@ private void createSender(Session session) {
       Source source = new Source();
       source.setAddress(linkInfo.getSourceAddress());
       source.setDurable(TerminusDurability.UNSETTLED_STATE);
+      source.setDefaultOutcome(new Released());
```

Comment:
Whilst it is not wrong to set the default-outcome, since omitting it implies Accepted (i think) and this works for our use-case, I would do that.  I'm just fearful of bugs in less trodden paths of the implementations.

Reply:
In case my comment wasn't clear, I am simply suggesting removing line 261

Reply:
Yes it was clear, just haven't gotten to it yet @k-wall  :)